app/controllers/manage/configs_controller.rb

```diff
@@ -62,6 +63,24 @@ def exit_theming_editor
     redirect_to manage_configs_path
   end
 
+  def reset_hackathon
+    data = params[:hackathonReset]
+    roles_to_delete =
+      { user: data[:users].to_i,
+        volunteer: data[:volunteers].to_i,
+        organizer: data[:organizers].to_i,
+        director: data[:directors].to_i }
+    roles_to_delete.each do |r, delete|
+      next if delete != 1
```

Comment:
```suggestion
      next if delete.zero?
```

https://stackoverflow.com/questions/3534709/ruby-integer-to-boolean